includes/options-page.php

```
@@ -909,7 +909,7 @@ function edac_sanitize_pro_summary_heading( $input ) {
  * @return int either 1 for checked or 0 for unchecked
  */
 function edac_sanitize_checkbox( $input ) {
-	return ( isset( $input ) && '1' === $input ) ? 1 : 0;
+	return ( isset( $input ) && ( '1' === $input || 1 === $input ) ) ? 1 : 0;
```

PHPStan (2.1.32)

Variable $input in isset() always exists and is not nullable.


(isset.variable)